customcommands/web.go

```diff
@@ -285,6 +281,27 @@ func handleGetCommand(w http.ResponseWriter, r *http.Request) (web.TemplateData,
 	return serveGroupSelected(r, templateData, cc.GroupID.Int64, cc.GuildID)
 }
 
+func handleGetPublicCommand(w http.ResponseWriter, r *http.Request) (web.TemplateData, error) {
+	_, templateData := web.GetBaseCPContextData(r.Context())
+
+	var cc *models.CustomCommand
+	var err error
+	cmdParam := pat.Param(r, "cmd")
+	// accessed via shortlink, interpret as public ID instead
+	cc, err = models.CustomCommands(
+		models.CustomCommandWhere.PublicID.EQ(cmdParam)).OneG(r.Context())
+	if err != nil {
+		templateData.AddAlerts(web.ErrorAlert("Command couldn't be found via shortlink"))
+		return templateData, errors.WithStackIf(err)
+	}
+	templateData["CC"] = cc
+	templateData["Commands"] = true
+	templateData["PublicID"] = cc.PublicID
```

Comment:
I'm having trouble identifying where this is used. Would you please share what it's used for?

Reply:
I was going to use this to replace .LocalID everywhere on the Public page, but then that wasn't needed later.